satellite/audit/reporter.go

```diff
@@ -167,6 +174,16 @@ func (reporter *Reporter) recordAuditSuccessStatus(ctx context.Context, successN
 			IsUp:         true,
 			AuditSuccess: true,
 		}
+
+		notification := &pb.Notification{
```

Comment:
Just for debugging, will move to failed and contained audits, to reduce network load

Reply:
add a proper TODO, so we don't forget about it.